contracts/solidity/src/YABTransfer.sol

```diff
@@ -73,15 +76,33 @@ contract YABTransfer is Initializable, OwnableUpgradeable, UUPSUpgradeable {
             payload);
     }
 
-    function setEscrowAddress(uint256 snEscrowAddress) external {
-        require(msg.sender == _owner, "Only owner can call this function.");
+    function setEscrowAddress(uint256 snEscrowAddress) external onlyOwner {
         _snEscrowAddress = snEscrowAddress;
     }
 
-    function setEscrowWithdrawSelector(uint256 snEscrowWithdrawSelector) external {
-        require(msg.sender == _owner, "Only owner can call this function.");
+    function setEscrowWithdrawSelector(uint256 snEscrowWithdrawSelector) external onlyOwner {
         _snEscrowWithdrawSelector = snEscrowWithdrawSelector;
     }
 
+    
+    //// MM ACL:
+
+    function getMMAddress() external view onlyOwnerOrMM returns (address) {
```

Comment:
Why is this restricted?

Reply:
I thought hiding a little bit the MM address couldn't harm. But i can totally remove the restriction (and change its related test)

Reply:
I think we should remove that restriction because the address is public. The onlyOwner is only a restriction when the function is called. A third party can check the storage and find the address without this function as well.